vault/resource_okta_auth_backend.go

```diff
@@ -288,8 +340,38 @@ func oktaAuthBackendRead(d *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 
+	if err := oktaReadAuthConfig(client, path, d); err != nil {
+		return err
+	}
+
 	return nil
+}
+
+func oktaReadAuthConfig(client *api.Client, path string, d *schema.ResourceData) error {
+	log.Printf("[DEBUG] Reading auth config for mount %s from Vault", path)
+	config, err := client.Logical().Read(oktaConfigEndpoint(path))
+	if err != nil {
+		return err
+	}
 
+	// map schema config to okta auth params.
+	fieldMap := map[string]string{
+		"organization": "organization",
+		"ttl":          "token_ttl",
+		"max_ttl":      "token_max_ttl",
```

Comment:
Do we need to add the other parameters here? Like `base_url`, `bypass_okta_mfa`, and `token`? Those come up as blank when I import an existing okta mount.

Reply:
Fixed in [4ce4298](https://github.com/hashicorp/terraform-provider-vault/pull/1123/commits/4ce429803284ec3d38a76fcaa1d165ee69ed0e28)